src/components/CoinsList.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-len */
```

Comment:
Just as requested by the previous reviewers, kindly take away `eslint-disable` as linters are present in your application to ensure you write better code😉. To fix this issue, I suggest you take a look at each line (e.g. line 20). The linter produces an error because the number of characters in the line is more than the defined number of characters for a line. you can simply split that line of code to take two lines so we have fewer characters per line😉.